bloom-housing/bloom
# 2328/address verification

## Issue Overview

This PR addresses #2328 


## Description

This PR adds one extra step to the public Application > Primary Applicant Address form to validate and confirm the address via the MapBox API.

## How Can This Be Tested/Reviewed?

Try filling out an application for a listing on the Public site, and on the second "contact" step try putting in both real addresses and garbled data to make sure the interface is reasonably robust to allow saving the new address (or at least what was entered). Also: https://github.com/bloom-housing/bloom/pull/2412#issuecomment-1015834594

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
